src/app/test/cases/agent/request-different-loader.case.ts

```diff
@@ -0,0 +1,30 @@
+import { assert } from '../../service/assert'
+import { TestCase } from '../../types/testCase'
+import { getApiKey } from '../../utils/getApiKey'
+
+const testCase: TestCase = {
+  name: 'agent request with different loaderVersion',
+  test: async (api) => {
+    const query = new URLSearchParams()
+
+    query.set('apiKey', getApiKey())
+    query.set('version', '3')
+
+    const loaderVersion = '3.9.8_test_worker'
```

Comment:
Can we change this case to `loader_v3.9.8_e1_noworker.js`, please?

Reply:
Done!